### PR TITLE
VACMS-18460 Remove unneeded custom GA events in Find Forms

### PR DESCRIPTION
**Pull request by randimays**
## Are you removing, renaming or moving a folder in this PR?
- [x] No, I'm not changing any folders (skip to Summary and delete the rest of this section)
- [ ] Yes, I'm removing, renaming or moving a folder

### :warning: TeamSites :warning:
Examples of a TeamSite: https://va.gov/health and https://benefits.va.gov/benefits/. This scenario is also referred to as the "injected" header and footer. You can reach out in the `#sitewide-public-websites` Slack channel for questions.

Did you change site-wide styles, platform utilities or other infrastructure?
- [x] No
- [ ] Yes, and I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#that-sounds-normal-so-whats-the-proxy-all-about) to test the injected header scenario

## Summary
Removal of unneeded custom Google Analytics events in the Find Forms application.

I also discovered that we are using `aria-label` for `<va-link>` in a couple spots where we should be using `label`. I corrected that while I was in here.

## Related issue(s)
- https://github.com/department-of-veterans-affairs/va.gov-cms/issues/18460

## Screenshots

Tested at `/discharge-upgrade-instructions`
<details><summary>DUW "Edit" link aria-label fix</summary>
Screenshot 2024-07-29 at 12 17 47 PM
Screenshot 2024-07-29 at 12 17 42 PM
</details>

Tested at `/health-care/refill-track-prescriptions`
<details><summary>CTA widget aria-label fix</summary>
Screenshot 2024-07-29 at 1 03 37 PM
Screenshot 2024-07-29 at 1 03 35 PM

</details>

Tested from `/find-forms` searching "1010"
<details><summary>BAKED-IN events for "Frequently used VA forms" links</summary>
Screenshot 2024-07-29 at 12 53 02 PM

#### File a VA disability claim
Screenshot 2024-07-29 at 12 19 24 PM
Screenshot 2024-07-29 at 12 21 32 PM

#### Apply for the GI Bill and other education benefits
Screenshot 2024-07-29 at 12 19 48 PM
Screenshot 2024-07-29 at 12 21 40 PM

#### Apply for VA health care
Screenshot 2024-07-29 at 12 19 58 PM
Screenshot 2024-07-29 at 12 21 48 PM

</details>

<details><summary>CUSTOM event for when search results are shown</summary>
Screenshot 2024-07-29 at 12 07 52 PM
</details>

<details><summary>BAKED-IN event for opening modal (clicking "Download VA Form ___" on search results page)</summary>
Screenshot 2024-07-29 at 12 53 28 PM

Screenshot 2024-07-29 at 12 09 38 PM
</details>

<details><summary>CUSTOM event for clicking a search result</summary>
Screenshot 2024-07-30 at 9 41 10 AM
Screenshot 2024-07-30 at 9 57 03 AM

</details>

<details><summary>BAKED-IN event for clicking a form's tool link</summary>
Screenshot 2024-07-30 at 9 38 59 AM
Screenshot 2024-07-30 at 9 39 44 AM

</details>

<details><summary>CUSTOM event for using "Sort by" on the search results page</summary>
Screenshot 2024-07-29 at 12 53 50 PM

Screenshot 2024-07-29 at 12 16 35 PM
</details>

### CODE REVIEW

File: src/applications/find-forms/components/SearchResult.jsx [hunk 1]
```diff
@@ -189,20 +182,14 @@ const SearchResult = ({
       {relatedTo}
       {relativeFormToolUrl ? (
         <div className="vads-u-margin-bottom--2p5">
-          <va-link
+          {/* va-link-action does not add the lang attribute; we can convert if this is addressed */}
```
Comment (randimays) [inline src/applications/find-forms/components/SearchResult.jsx:185]: Changed this back to `<a>`. It was a `<va-link>` with the old classname on it; likely a mistake when we converted. With that setup it was showing as a plain link. Now it's back to showing the old action link version. "Fill out VA Form 1010-EZR online" link below is how it currently looks with this PR.

Screenshot 2024-07-29 at 12 52 16 PM

**Comment by randimays** [comment]
@laflannery Holding this for your review; I sent ya an email about `lang` attributes on `<va-link>` (and `<va-link-action>`) not being supported on the web components. Wanting to make sure we definitely need those `lang` attributes -- and if so, I'll keep the links in question as plain `<a>` tags until we get that support (and I'll create a DST issue for it).

Title of the form and action link
Screenshot 2024-07-29 at 12 52 16 PM

**Review by chriskim2311 — DISMISSED** [review]
LGTM!

**Comment by laflannery** [comment]
@randimays I looked up [a Spanish form](https://www.va.gov/find-forms/?q=Form+21P-0519C-1) and we only have detail pages for the English versions, not the Spanish ones. Therefore, these don't need the `lang` attribute at this time and can be switched to use the `va-link`

Regarding the `action` links, these are only on the forms that have form tools within the VA ecosystem and we don't have any Spanish online tools at this time. However, in your review instance these are Primary action links and they should be the blue secondary because there should only be 1 primary action on each page.

**Comment by laflannery** [comment]
@randimays I other thing I just noticed, when I click the new action links all of them are taking me to a 404 page, is that right? I'm not sure if it's because it's a RI or if there is an issue

**Comment by randimays** [comment]
@laflannery I checked some of the links that lead to the form's tool page in the review instance and they 404'd as well. I tried those links in prod and they have redirects on them (which wouldn't exist in lower environments), so that explains the 404s assuming the case is the same for the ones you checked.

**Review by micahchiang — APPROVED** [review]

**Comment by randimays** [comment]
@chriskim2311 @laflannery this is ready for your re-review please!

**Review by laflannery — APPROVED** [review]
Looks good, also tested Very large font and it all scaled correctly!

**Review by chriskim2311 — APPROVED** [review]
LGTM!

**Pull request merged**